src/Stratis.Bitcoin.Features.Consensus/Rules/CommonRules/BitcoinHeaderVersionRule.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using NBitcoin;
+using Stratis.Bitcoin.Base.Deployments;
+using Stratis.Bitcoin.Consensus.Rules;
+using Stratis.Bitcoin.Utilities;
+
+namespace Stratis.Bitcoin.Features.Consensus.Rules.CommonRules
+{
+    /// <summary>
+    /// Checks if <see cref="BitcoinMain"/> network block's header has a valid block version.
+    /// </summary>
+    [HeaderValidationRule(CanSkipValidation = true)]
+    public class BitcoinHeaderVersionRule : HeaderVersionRule
```

Comment:
Shouldn't this rule be merged with BitcoinActivationRule?  It looks like they are logically very close